apps/angular/src/core/services/anime.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Anime } from '@js-camp/core/models/anime';
+import { Observable, delay, map } from 'rxjs';
+import { environment } from '@js-camp/angular/environments/environment';
+import { AnimeDto } from '@js-camp/core/dtos/anime.dto';
+import { Pagination } from '@js-camp/core/models/pagination.dto';
+import { PaginationDto } from '@js-camp/core/dtos/pagination.dto';
+import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
+import { AnimeMapper } from '@js-camp/core/mappers/anime.mapper.';
+import { AnimeQueryParams } from '@js-camp/core/models/anime-query-params';
+import { AnimeQueryParamsMapper } from '@js-camp/core/mappers/anime-query-params.mapper';
+
+/** Anime service. */
+@Injectable({
+	providedIn: 'root',
+})
+export class AnimeService {
+	public constructor(private http: HttpClient) { }
```

Comment:
Please don't forget about readonly

```suggestion
	public constructor(private readonly http: HttpClient) { }
```